dcm4che-tool/dcm4che-tool-storescu/src/main/resources/org/dcm4che3/tool/storescu/messages.properties

```diff
@@ -27,6 +27,7 @@ tmp-file-dir=directory were temporary file with File Meta Information from scann
 if not specified, the file is stored into the default temporary-file directory
 tmp-file-prefix=prefix for generated file name for temporary file; 'storescu-' by default
 tmp-file-suffix=suffix for generated file name for temporary file; '.tmp' by default
+input-file=file containing paths of files to be sent instead of reading them as arguments from the command line
```

Comment:
Please also re-generate the README.md file in dcm4che-tool/dcm4che-tool-storescu/